pom.xml

```diff
@@ -140,6 +140,11 @@
             <artifactId>lombok</artifactId>
         </dependency>
         <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+            <version>1.10</version>
```

Comment:
we don't want to add any dependencies to the project. we should implement Base64 ourselves, just like we did it with `CcHex`